eng/pipelines/templates/jobs/archetype-sdk-client-samples.yml

```diff
@@ -36,8 +36,6 @@ stages:
           version: '$(go.version)'
         displayName: "Select Go Version"
 
-      - template: ../steps/create-go-workspace.yml
```

Comment:
Should we delete this template as well?

Reply:
It's still used by a cosmos pipeline that I think needs it...